packages/fluentui/react-northstar/src/themes/teams/staticStyles/globalStyles.ts

```diff
@@ -7,6 +7,7 @@ const globalStyles: StaticStyleFunction = siteVars => ({
     fontFamily: siteVars.bodyFontFamily,
     fontSize: siteVars.bodyFontSize,
     lineHeight: siteVars.bodyLineHeight,
+    outline: 'none',
```

Comment:
I would like to fix this issue in the exact component instead of global styles.

Reply:
@layershifter - It's on almost every element. Normally I would agree, but this is probably more efficient and cleaner?

In Teamspace-Web, we have this:
![image](https://user-images.githubusercontent.com/6237759/84959527-5aab5080-b0b4-11ea-97d6-3e9781a58cb1.png)

Reply:
@layershifter  - what do you think?

Reply:
Can we at least move this style to `Chat` component itself? I.e. will affect `Chat` and everything inside instead of fully global space.

Reply:
@notandrew can we update the PR based on the proposed comment from @layershifter ? I think it make sense that these "global" styles are moved at least to `chatStyles.ts`. Note that you can use these selectors there too:

```js
"& *": { outline: 'none'} 
```